StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/NamingRules/SA1308CodeFixProvider.cs

```diff
@@ -66,16 +60,24 @@ public override FixAllProvider GetFixAllProvider()
                     break;
                 }
 
-                if (!string.IsNullOrEmpty(token.ValueText))
+                if (numberOfCharsToRemove == 0)
+                {
+                    continue;
```

Comment:
❓ I noticed from the code coverage report that this `continue` statement isn't hit by the current tests. Is it possible to reach this `if` statement and still have `numberOfCharsToRemove` be 0?